roles/chrony/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+chrony_package_name: chrony
+chrony_service_name: chronyd
+chrony_config_file: /etc/chrony.conf
+chrony_driftfile: /var/lib/chrony/drift
+chrony_logdir: /var/log/chrony
+
+chrony_servers:
+  - pool.ntp.org iburst
```

Comment:
I see where you are going with this, but I don't think that the variables in our ansible configuration need to use the same names as in the configuration file.  For variables that are "a list of servers", I like the `_servers` suffix.  I guess I don't really feel strongly about that, so I'll go ahead and make this change.